soc/xtensa/nxp_adsp/imx8ulp/include/soc/memory.h

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (c) 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_MEMORY_H
```

Comment:
Change the include guard to the standard ZEPHYR_

Reply:
cleaned up the file.